content/docs/dvclive/how-it-works.md

```diff
@@ -81,13 +81,17 @@ find but that don't clutter your Git history or create extra branches.
 ### Track large artifacts with DVC
 
 Models and data are often large and aren't easily tracked in Git.
-`Live.log_artifact("model.pt")` will
+`Live.log_artifact("model.pt", type="model")` will
 [cache](/doc/start/data-management/data-versioning) the `model.pt` file with DVC
 and make Git ignore it. It will generate a `model.pt.dvc` metadata file, which
 can be tracked in Git and becomes part of the experiment. With this metadata
 file, you can [retrieve](/doc/start/data-management/data-versioning#retrieving)
 the versioned artifact from the Git commit.
 
+Passing `type="model"` or `type="data"` will add it to `artifacts` section of
+`dvc.yaml`, allowing DVC to understand what it is and show models in
+[Studio Model Registry](/doc/use-cases/model-registry).
```

Comment:
This still feels a little confusing to me because:
1. We aren't currently working on any Studio functionality for `type=data` (let's only talk about plans for things that are at least in progress already).
2. It still sounds like it only adds to `artifacts` if `type=model` or `type=data`, which isn't true.

Reply:
Hmm, sorry - I thought I fixed that. Doing it now.

Reply:
Ok, did that. It's ready to merge I believe.